src/xo-app/sr/tab-advanced.js

```diff
@@ -45,7 +45,7 @@ const UnhealthyVdiChains = flowRight(
 )(({ chains, vdis }) => isEmpty(vdis)
   ? null
   : <div>
-    <h3>{_('srUnhealthyVdiTitle')}</h3>
+    <h3>{_('srUnhealthyVdiTitle', { total: sum(values(chains)) })}</h3>
```

Comment:
I thought it was a bit overkill to make it a stateful component just for this small sum. Should I do it anyway?